### ISSUE TITLE
Some UI "links" should be "buttons" (e.g. MyBooks, Catalogs, Settings)

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#598]

**Comment by danielweck** [comment]
I am wrong, sorry. The links I identified in fact have the correct semantics (they should not be buttons). Closing for now, will re-open if I find concrete examples of mismatch.
